frontend/workflows/projectCatalog/src/details/index.tsx

```diff
@@ -126,7 +126,7 @@ const Details: React.FC<ProjectDetailsWorkflowProps> = ({ children, chips }) =>
     <ProjectDetailsContext.Provider value={projInfo}>
       <StyledContainer container direction="row" wrap="nowrap">
         {/* Column for project details and header */}
-        <Grid item direction="column" xs={12} sm={12} md={12} lg={12} xl={12}>
+        <Grid container item direction="column" xs={12} sm={12} md={12} lg={12} xl={12}>
```

Comment:
Should the children of this have `item` props if this is a container?

Reply:
I dont think it's technically necessary if we're not using spacing but I've added it just for consistency.